sdk/identity/azure-identity/CMakeLists.txt

```diff
@@ -26,11 +26,6 @@ if(NOT AZ_ALL_LIBRARIES)
   endif()
 endif()
 
-if(BUILD_TESTING)
-  # define a symbol that enables some test hooks in code
-  add_compile_definitions(TESTING_BUILD)
-endif()
```

Comment:
Why are we removing this?

Reply:
No code depends on `TESTING_BUILD` macro in identity.